src/command/sample_encode/cache.rs

```diff
@@ -17,6 +17,7 @@ pub async fn cached_encode(
     input_size: u64,
     full_pass: bool,
     enc_args: &FfmpegEncodeArgs<'_>,
+    vmaf_args: &impl Hash,
```

Comment:
Minor: Could just use `&Vmaf` type here as i don't think we need the generic flexibility